src/lessons/08_grid_layout/grid-layout.slides.html

```diff
@@ -0,0 +1,758 @@
+               }
+               .grid-item-1 {
+                grid-column: 1 / 4;
+               }
+               .grid-item-2 {
+                grid-row: 2 / 4;
+               }
+
+               @media only screen and (max-width: 1000px){
+                 .grid-container{
+                  grid-template-columns: 1fr 1fr;
+                 } 
+                 .grid-item-1 {
+                  grid-column: 1 / 3;
+                  grid-row: 2 / 3;
+                 } 
+                 .grid-item-2 {
+                  grid-row: 1 / 1;
+                 }
+               }
+  
+            # HTML
+            <div class="grid-container">
+              <div class="grid-item-1">Item 1</div>
+              <div class="grid-item-2">Item 2</div>
+              <div class="grid-item-3">Item 3</div>
+              <div class="grid-item-4">Item 4</div>
+              <div class="grid-item-5">Item 5</div>
+            </div></code><br>
+            <div class="grid-container grid-16">
+              <div class="grid-item-1">Item 1</div>
+              <div class="grid-item-2">Item 2</div>
+              <div class="grid-item-3">Item 3</div>
+              <div class="grid-item-4">Item 4</div>
+              <div class="grid-item-5">Item 5</div>
+            </div>
+            </pre>
+            </section>
+
+            <section>
+                <h3>Some Best Practices</h3>
+                <ul>
+                    <li>Use grid with flexbox, not instead of it</li>
+                    <li>Take advantage of negative line numbers</li>
+                    <li>Name your grid lines</li>
+                </ul>
+            </section>
+
+            <section>
+                <h3>Homework</h3>
+                <p>The "ReDI News" team wants to setup an online version of their newspaper. They ask you the help them with this task. As they would like to reach as many readers as possible, the page should be easily readable on a laptop and phone screen.</p>
+            </section>
+
+            <section>
+                <h3>Homework</h3>
+                <p>The team would like you to create a single-page application, the start page, which comprises the following content:
+                    <ul>
+                        <li>a header with the name of the Newspaper.</li>
+                        <li>one section for a preview of the featured article of the day</li>
+                        <li>three sections for the previews of three secondary articles of the day</li>
+                        <li>a navigation bar</li>
+                        <li>a footer</li>
+                    </ul>
+                    They also give you the freedom to add additional sections which you would like to include.</p>
+            </section>
+
+            <section>
+                <h3>Homework</h3>
+                <ol>
+                    <li>Create a mockup for the laptop version</li>
+                    <li>Implement your page design with CSS Grid</li>
```

Comment:
Let's not forget to clarify what a mockup is and provide them with ideas on how to approach them

Reply:
@Stephs15 Would you suggest that they use a dedicated tool like Sketch? Maybe that will be too much for a simple design. Or do you mean like what to think about when creating a mockup?

Reply:
I wouldn't even go that far. A drawing on a piece of paper should be good enough for starters